ani-cli

```diff
@@ -205,13 +218,13 @@ search="${ANI_CLI_DEFAULT_SOURCE:-scrape}"
 while [ $# -gt 0 ]; do
 	case "$1" in
 	--vlc) player_function=vlc && shift ;;
-	-d) player_function=download && shift ;;
-	-s) player_function=syncplay && shift ;;
-	-c) search=history && shift ;;
-	-q) quality="$2" && shift 2 ;;
+	-s | --syncplay) player_function=syncplay && shift ;;
+	-q | --quality) quality="$2" && shift 2 ;;
+	-c | --continue) search=history && shift ;;
+	-d | --download) player_function=download && shift ;;
 	-v | -V | --version) version_info ;;
```

Comment:
I'd actually prefer if -v and -V would be used for vlc, as I think that function is used more frequently. Also I'd consider supporting case insensitive & long arguments for all other options, bc it requires no extra loc (but I'm open on this)

Reply:
good idea for vlc, will change that in the next commit. added long options in 3b19f3aa202f7877ca27bbe92ef2eb214a38ddd8

Reply:
Especially `-v` needs to me vlc. 
`-V` is conventionally a shorthand for `--version`.
Would also make sense with the heuristic that uppercase makes stuff outside the script.